package sec1;
public class UserException1 {
	public static void main(String[] args) {
		method1();
	}
	public static void method1() throws RunException { //throw 자동으로 예외를 발생시킬때
		throw new RunException("사용자 예외");
		
	}

}
